source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,243 @@
+- :java-core-api:`TextSearchOptions <com/mongodb/client/model/TextSearchOptions.html>`
+
+.. _term_search:
+
+Search Text By a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to search for
+a term in your text search. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. The ``Filters.text()`` method performs a logical OR on
+each term and returns documents that contain any of the terms. 
+
+Example 
+```````
+
+The student wants to find Science courses with case sensitivity on
+the first letter being capital.
+
+To address this scenario, the student performs the following:
+
+- Specify to have the search be case sensitive though ``TextSearchOptions``
+- Perform a text search on documents that contain the term "Science" with the ``Filters.text()`` method
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Mathematical Structures for Computer Science", "major": ["Computer Science", "Mathematics"] }
+   { "_id": 3, "name": "Environmental Science", "major": ["Environmental Science", "Sustainable Management"] }
+
+.. note::
+
+   The output would consist of all the documents in the
+   ``junior_courses`` collection if the search was not case sensitive
+   because each document contains the term "Science" in the ``name``
+   field. 
+
+Search Text By a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+search for a phrase in your text search. If you don't add escaped quotes,
+the method performs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The student wants to find Computer Science courses.
+
+To address this scenario, the student performs a text search on
+documents that contain the phrase "Computer Science":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 2, "name": "Discrete Mathematical Structures for Computer Science", "major": ["Computer Science", "Mathematics"], "score": 1.2 }
+   { "_id": 4, "name": "Image Processing in Computer science", "major": ["Artificial Intelligence", "Computer Science"], "score": 1.25 }
+ 
+Search Text With Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefix a minus sign ``-`` to each term you want to exclude in your text
+search. 
+
+Example 
+```````
+The student wants to only find Science courses.
+
+To address this scenario, the student performs a text search on
+documents that contain the term "Science", but do not contain the term
```

Comment:
I would make sure it's clear that you're searching on a collection for documents that contain some value in a field.